db/migrations/20240624201152_AddPasswordHashNotNull.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE users 
+DROP COLUMN password_hash;
```

Comment:
updated the migration to make the `password_hash` not nil

Reply:
I edited this migration so that instead of dropping the columm here we set it to nullable!

Because the idea of the `down` migration is to reverse the `up`, and since the up didn't create the column we dont want to drop it here!